internal/mode/static/telemetry/collector.go

```diff
@@ -214,3 +219,14 @@ func CollectClusterID(ctx context.Context, k8sClient client.Reader) (string, err
 	}
 	return string(kubeNamespace.GetUID()), nil
 }
+
+// CollectImageSource gets the source of the NGF image. This is done by inspecting the BUILD_AGENT environment variable.
+// Valid sources are: "gha" for images built using GitHub Actions in the pipeline, or "local".
+// If the environment variable is not set to one of these, the source is "unknown"
+func CollectImageSource() string {
```

Comment:
since this function is not used outside of the package, let's keep it private

Reply:
Removed with the previous change 😄